frontend/src/metabase-lib/expressions/suggest.js

```diff
@@ -16,11 +16,7 @@ import {
   formatSegmentName,
 } from "./index";
 
-const suggestionText = func => {
-  const { displayName, args } = func;
-  const suffix = args.length > 0 ? "(" : " ";
```

Comment:
I tagged Ariya as reviewer but I don't see empty space `" "` scenario covered in your solution.

Reply:
Thanks @nemanjaglumac 

Ariya's review will be most welcome.

You ended up giving me an idea.

Since zero-arity functions should be followed by opening-then-closing-parentheses, that is, `()`, I updated the autocomplete to add the whole thing.

Reply:
It seems like the `" "` scenario is there because `COUNT` and `CUMULATIVECOUNT` have the `()` as optional.

See these PRs from Ariya for context:
https://github.com/metabase/metabase/pull/20432
https://github.com/metabase/metabase/pull/20533

I've raised the question on what we should do [here](https://metaboat.slack.com/archives/C03SHMZ266R/p1668676917056199)

Reply:
Bruno suggests we leave it as is. I'll close this PR, sorry for the trouble

Reply:
@calherries, I'm curious: where are you using `COUNT` and `CUMULATIVECOUNT`?

Asking because the context I was able to use them in (the native query editor), autocomplete differently.